hosting/src/app/games/quiz/lobby/lobby.component.ts

```diff
@@ -48,6 +48,8 @@ export class LobbyComponent implements OnInit, OnDestroy {
                     ...rounds,
                 ];
                 this.availableRounds = this.rounds.map((round) => ({ id: round.uid, title: round.title }));
+                let roundNumber = this.availableRounds.length;
+                this.questionForm.controls[ 'round' ].setValue(roundNumber > 0 ? this.availableRounds[ roundNumber - 1 ] : undefined);
```

Comment:
I am unsure if this would be behaviour we want, my reason being there are 2 forms on the page, as a user if I add a question, then realise the round I wanted I've not yet made, it would update the form.... also the first time I add a question im not going to become alerted to needing to pick the round wanted....

This needs a bit more user feedback I think before we should change its behaviour....

Reply:
I've added added a variable to prevent the accidental change of round by any other quiz editor creating a new round in the middle of a quiz/question edition.

Please just let me explain my stance in favor of the round preselection with a bit more of detail. If you still disagree, I am happy to adjust it to just run on new questions and keep the rest for my personal enjoyment. no worries at all.

Preloading round on entering edition and new round creation aims to ease the most usual cases:
-If I was working in a quiz, paused and retake the job, the most likely is that I would keep adding question in the last round.
-If I  am writing a question for a quiz and realize "this goes in a round I haven't created yet", the round would be updated for me. If the question should be in the current round, I think I would finish editing it first.
-If I finish a question and I want to create a new round to start adding questions to it,  it would save my time on selecting it.

In any case the select is there to adjust to the round as needed. 
Worst case scenario is same amount of clicks thank without preselection. 
Best case, 2 clicks less per question.

I think it is a good deal. While developing and testing I can ensure saves lots of time.

Reply:
Point 1 I agree (I had actually done this already on some local code anyhow)

Point 2 and 3 my biggest concern is potential side effects - 2 people adding questions a round gets added and the subscribe triggers a round selection change on both sides.... while you can code around this I don't know that the benefit is high enough, as round selection is the first choice (most people) follow a form and would be thinking "this is for round x" and if its missing add the round there and then, which brings to the final point (3) where I am on the fence as in that last scenario I can see that "perhaps" I want to change the selection. But equally I can see cases I don't want this.

I shall have a play locally later today and make a decision... its borderline but I'm thinking in terms of side effects and other things coming in the future what this impact can be